otter/convergence/selfheal.py

```diff
@@ -126,7 +126,8 @@ def check_and_trigger(tenant_id, group_id):
                   tenant_id=tenant_id, scaling_group_id=group_id)
     else:
         state = info["state"]
-        if state.status == ScalingGroupStatus.ACTIVE and (not state.paused):
+        if (state.status == ScalingGroupStatus.ACTIVE and
+                (not state.paused) and (not state.suspended)):
```

Comment:
Dumb nitpick, but this might read more nicely as:

`(state.status == ScalingGroupStatus.ACTIVE and not (state.paused or state.suspended))`